integration/spark/src/main/scala/org/apache/spark/util/AlterTableUtil.scala

```diff
@@ -1191,4 +1191,28 @@ object AlterTableUtil {
     }
   }
 
+  def revertSchemaInCaseOfFailure(carbonTable: CarbonTable, deleteCols: Seq[ColumnSchema],
```

Comment:
change method name and add the method level comment

Reply:
done